web/massastation/src/pages/store/MyPlugin.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+
+import {
+  Button,
+  Certificate,
+  MassaWallet,
+  Plugin,
+} from '@massalabs/react-ui-kit';
+import { FiArrowUpRight, FiRefreshCcw, FiTrash2 } from 'react-icons/fi';
+import { IMassaPlugin } from './MyStation';
+
+enum PluginStatus {
+  Up = 'Up',
+  Down = 'Down',
+}
+
+function MyPlugin({ plugin }: { plugin: IMassaPlugin }) {
+  const [myPlugin, setMyPlugin] = useState<IMassaPlugin>(plugin);
+  function handlePluginState(method: string) {
+    // TODO :
+    // const {mutate } = usePost<any>("plugin-manager");
+    // const ok = mutate({method:"${method}"})
+    const newStatus = method === 'start' ? PluginStatus.Up : PluginStatus.Down;
+    const updatedPlugin = { ...myPlugin, status: newStatus };
+    setMyPlugin(updatedPlugin);
+  }
+  const argsOn = {
+    preIcon:
+      myPlugin.author === 'MassaLabs' ? (
+        <MassaWallet variant="rounded" />
+      ) : (
+        <img src={myPlugin.logo} />
+      ),
+    topAction: (
+      <Button onClick={() => handlePluginState('stop')} variant="toggle">
+        on
+      </Button>
+    ),
+    title: `${myPlugin.name} `,
+    subtitle: `${myPlugin.author}`,
+    subtitleIcon: myPlugin.author === 'MassaLabs' ? <Certificate /> : <></>,
+    content: [
+      // conditionally render the update icon based on the updatable property
```

Comment:
?

Reply:
Certification badge if the authos is Massa Labs

Reply:
Remove the comment.

Hard to understand what means Updatable. Can we pick another explicit name!?

Reply:
I'm not sure @Pierregvx ... but I have a hard time imagining something more explicit than "updatable" to signify that an update is possible ? I will take any suggestions :)